packages/cli/generators/service/templates/service-template.ts.ejs

```diff
@@ -0,0 +1,24 @@
+import {getService, juggler} from '@loopback/service-proxy';
+import {inject, Provider} from '@loopback/core';
+import {<%= dataSourceClassName %>} from '../datasources';
+
+/**
+ * this is where you define the Node.js methods that will be
```

Comment:
Could you please move this comment into the class and also don't use `/**` because that's for `tsdoc`.

```
export interface <%= className %>Service {
  // Declare methods provided by the remote service here.
  // These methods should map 1:1 to operations configured 
  // in and provided by the datasource (e.g. SOAP operations).
}
```